core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -382,7 +390,72 @@ private FormValidation load(String json, long dataTimestamp) throws IOException
 
         @Restricted(NoExternalUse.class)
         public FormValidation updateNow() throws IOException {
-            return load(loadJSONHTML(new URL(getUrl() + ".html?id=" + URLEncoder.encode(getId(), "UTF-8") + "&version=" + URLEncoder.encode(Jenkins.VERSION, "UTF-8"))), System.currentTimeMillis());
+            List<JSONObject> jsonList = new ArrayList<>();
+            for (String site : getUrls()) {
+                String jsonString;
+                try {
+                    jsonString = loadJSONHTML(new URL(site + ".html?id=" + URLEncoder.encode(getId(), "UTF-8") + "&version=" + URLEncoder.encode(Jenkins.VERSION, "UTF-8")));
+                } catch (Exception e) {
+                    LOGGER.warning("Could not load json from " + site);
```

Comment:
yesh but that was old code : https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/DownloadService.java#L385